controllers/controllers.js

```diff
@@ -1,9 +1,17 @@
-const {readTopics} = require('../models/models')
+const { request, response } = require("../app");
+const { readTopics } = require("../models/models");
+const fs = require("fs/promises");
 
 exports.getTopics = (request, response, next) => {
-    readTopics().then((topicRows) => {
-        response.status(200).send(topicRows)
-    })
-
-}
+  readTopics().then((topicRows) => {
+    response.status(200).send(topicRows);
+  });
+};
 
+exports.getEndpoints = (request, response, next) => {
+  fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then(
+      (fileContents) => {
+      response.status(200).send(JSON.parse(fileContents));
+    }
+  );
+};
```

Comment:
Good job deciding not to need a model here, because you don't! You just need a controller, nice work!

I would however send your fileContents over on an object with a key of endpoints or something!